### ISSUE TITLE
Bug fix for `setup.sh` go install

### ISSUE THREAD

**Issue by marissaeinhorn** [dev4vater/vater#188]
Moved source and path commands before checking `go version`

**Comment by marissaeinhorn** [comment]
Bug need to run `source /etc/path` to get path variable to change and include go

**Comment by marissaeinhorn** [comment]
Restarting semaphore service executes with error:
```
vater restart -s semaphore
[sudo] password for control:
Error: No such image: ansiblesemaphore/semaphore:dynamicVars
Cloning into '/home/control/vater/control-services/images/semaphore/build/src/github.com/ansible-semaphore/semaphore'...
remote: Enumerating objects: 9942, done.
remote: Counting objects: 100% (1002/1002), done.
remote: Compressing objects: 100% (225/225), done.
remote: Total 9942 (delta 838), reused 804 (delta 776), pack-reused 8940
Receiving objects: 100% (9942/9942), 6.49 MiB | 5.24 MiB/s, done.
Resolving deltas: 100% (6922/6922), done.
remote: Enumerating objects: 141, done.
remote: Counting objects: 100% (59/59), done.
remote: Total 141 (delta 59), reused 59 (delta 59), pack-reused 82
Receiving objects: 100% (141/141), 774.71 KiB | 2.65 MiB/s, done.
Resolving deltas: 100% (74/74), completed with 30 local objects.
From https://github.com/rmfirth/semaphore
 + ae7c867...4d3a74a develop           -> origin/develop  (forced update)
 * [new branch]      add_sshpass_to_dockerfile -> origin/add_sshpass_to_dockerfile
 * [new branch]      demo              -> origin/demo
 * [new branch]      gh-pages          -> origin/gh-pages
 * [new branch]      rfirth/addDynVars -> origin/rfirth/addDynVars
 * [new branch]      rfirth/cherryPick -> origin/rfirth/cherryPick
Switched to a new branch 'rfirth/cherryPick'
[task next ( deps )]
No matches.
Traceback (most recent call last):
  File "/home/control/vater/control-services/cli/vater.py", line 143, in <module>
    main()
  File "/home/control/vater/control-services/cli/vater.py", line 24, in main
    restart(c, p.args)
  File "/home/control/vater/control-services/cli/vater.py", line 87, in restart
    s.restartContainer()
  File "/home/control/vater/control-services/cli/semaphore.py", line 256, in restartContainer
    self.buildSemaphore()
  File "/home/control/vater/control-services/cli/semaphore.py", line 96, in buildSemaphore
    subprocess.check_output(
  File "/usr/lib/python3.8/subprocess.py", line 415, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/usr/lib/python3.8/subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['task', 'deps']' returned non-zero exit status 1.
```

**Comment by marissaeinhorn** [comment]
Testing on another vm, `task` installed properly, but script fails on 
```
  File "/home/control/vater/control-services/cli/vater.py", line 143, in <module>
    main()
  File "/home/control/vater/control-services/cli/vater.py", line 24, in main
    restart(c, p.args)
  File "/home/control/vater/control-services/cli/vater.py", line 87, in restart
    s.restartContainer()
  File "/home/control/vater/control-services/cli/semaphore.py", line 256, in restartContainer
    self.buildSemaphore()
  File "/home/control/vater/control-services/cli/semaphore.py", line 116, in buildSemaphore
    subprocess.check_output(
  File "/usr/lib/python3.8/subprocess.py", line 415, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/usr/lib/python3.8/subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['sudo', 'context=prod', 'tag=dynamicVars', 'task', 'docker:build']' returned non-zero exit status 1
```

**Comment by marissaeinhorn** [comment]
`root` users path didn't contain `go` within the path

**Comment by marissaeinhorn** [comment]
Research alternative methods for setting users path variables and ensure `setup.sh` doesnt add PATH commands to '/etc/profile' more than once

**Comment by marissaeinhorn** [comment]
Workaround is to run `sudo context=prod tag=dynamicVars task docker:build` inside of semaphore image location.  When running `vater clean -s semaphore` it will remove the docker image and require you to rerun the docker build command

**Comment by marissaeinhorn** [comment]
Closed by #190, lingering issues detailed in #191
